### PR TITLE
Tests: BZ#1220504 (usermod -p allowing colon (ie. '' )

### PR DESCRIPTION
**Pull request by aborah-sudo**
https://bugzilla.redhat.com/show_bug.cgi?id=1220504

### CODE REVIEW

File: multihost_test/Bugzillas/test_regression.py [hunk 1]
```diff
@@ -62,3 +62,38 @@ def test_bz_639975(self, multihost):
...
+          2. Should succeed
+          3. Should not succeed
+        """
+        user = "bz1220504"
+        client = multihost.client[0]
+        client.run_command("cp -vf /etc/shadow /etc/shadow_anuj")
+        client.run_command(f"useradd {user}")
+        # Check if the usermod -p does report error when pasword contains : (colon)
+        with pytest.raises(subprocess.CalledProcessError):
+            client.run_command(f"usermod -U {user}")
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_regression.py:88]: The comment (usermod -p) and the actual command executed (usermod -U) don't match.

**Review by ikerexxe — CHANGES_REQUESTED** [review]
I don't think this test is still valid, at least in its current version. Nowadays, the passwords are stored encrypted, thus you will not find the `:` character in `/etc/shadow` even if you were able to input it in the `usermod` command. Maybe there's some way of guessing what input characters may render `:` as an output when encrypted, but you will have to consult the crypto team.

IMHO, I'd reduce the test by removing all the checks in `/etc/shadow` and just check that `usermod -p ...:...` fails.

File: multihost_test/Bugzillas/test_regression.py (hunk 1 above)
Comment (aborah-sudo) [reply multihost_test/Bugzillas/test_regression.py:88]: I was porting this script :

https://pkgs.devel.redhat.com/cgit/tests/shadow-utils/tree/Regression/bz1220504-usermod-p-allowing-colon-ie-in-encrypted/runtest.sh

This is original script .

Comment (ikerexxe) [reply multihost_test/Bugzillas/test_regression.py:88]: That seems like a general comment for the complete test, so I'd suggest you to move it before the test in the python code.

Comment (ikerexxe) [reply multihost_test/Bugzillas/test_regression.py:88]: That seems like a general comment for the complete test, so I'd suggest you to move it before the test in the python code.

**Review by ikerexxe — APPROVED** [review]
LGTM

**Pull request merged**